staging/src/k8s.io/client-go/rest/with_retry.go

```diff
@@ -57,36 +57,33 @@ type WithRetry interface {
 	// A zero maxRetries should prevent from doing any retry and return immediately.
 	SetMaxRetries(maxRetries int)
 
-	// NextRetry advances the retry counter appropriately and returns true if the
-	// request should be retried, otherwise it returns false if:
+	// IsNextRetry advances the retry counter appropriately
+	// and returns true if the request should be retried,
+	// otherwise it returns false if:
 	//  - we have already reached the maximum retry threshold.
 	//  - the error does not fall into the retryable category.
 	//  - the server has not sent us a 429, or 5xx status code and the
 	//    'Retry-After' response header is not set with a value.
 	//
-	// if retry is set to true, retryAfter will contain the information
-	// regarding the next retry.
+	// IsNextRetry internally maintains the retry after
+	// parameters - retry reason, and wait duration associated
```

Comment:
I don't understand this part of the comment - what parameters? Where those are?

Reply:
yeah, it's confusing, i moved the comments below where it's more localized